Source/WebKit/UIProcess/API/Cocoa/WKPreferencesPrivate.h

```diff
@@ -236,6 +236,10 @@ typedef NS_ENUM(NSInteger, _WKPitchCorrectionAlgorithm) {
 @property (nonatomic, setter=_setAppNapEnabled:) BOOL _appNapEnabled WK_API_AVAILABLE(macos(10.15));
 #endif
 
+#if TARGET_OS_IPHONE
+@property (nonatomic) BOOL tabFocusesLinks WK_API_AVAILABLE(ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
```

Comment:
Targets that build from the same Xcode project, should be able to share implementation details without exposing them as SPI. Were you able to look into whether the cross-import target can `import WebKit_Internal`? I still totally believe that it can, so if I'm wrong I'd like to understand why.

Reply:
I forgot to; I will now

Reply:
I confirmed this is not possible; I get an error "No such module 'WebKit_Internal'"

Reply:
Sorry to be picky about this, but this isn’t a good enough reason. I believe it’s possible—modules are just files that the compiler has to discover on a search path. And avoiding needless SPI exports is a good goal.

Did you try setting SWIFT_INCLUDE_PATHS to include the path to WebKit/Modules/Internal? WebKit.xcconfig does this. You could probably just move this setting to BaseTarget.xcconfig since we want it to work from other targets:

```
SWIFT_INCLUDE_PATHS = $(SRCROOT)/Modules/Internal;
```